### DISCUSSION TITLE
airflow workers failed to connect redis. pod log of workers:

### DISCUSSION THREAD

**Discussion by robin202301** [apache/airflow#36717] [General]
airflow workers failed to connect redis. pod log of workers:
airflow helm chart version is 1.11.0

[2024-01-10 06:30:17,156: ERROR/MainProcess] consumer: Cannot connect to redis://:**@airflow-redis:6379/0: Error -2 connecting to airflow-redis:6379. Name or service not known..
Trying again in 32.00 seconds... (16/100)

_Originally posted by @robin202301 in https://github.com/apache/airflow/discussions/36706_

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
This is not airflow issue - you have problem with yor DNS resolving the name (so it looks like your deploymemt issue)  . Please don't open issues for troubleshooting of your deployment/K8s configuration . You need ot check why your DNS does not work - often it's a problem with resources.
